pyroclast/prototype/model.py

```diff
@@ -140,3 +146,11 @@ def conv_prototype_objective(self, min_distances, label=None):
 
         term_dict['l1'] = tf.norm(self.classifier.trainable_weights[0], 1)
         return term_dict
+
+    def features(self, x):
+        """Defined for implementing FeatureClassifierMixin"""
+        return self.prototype_activations(x)[0]
+
+    def classify_features(self, x):
```

Comment:
This should take as input `f`, not `x`. May be confusing in the future for someone.